src/components/Integrations/Shopify/Shopify.js

```diff
@@ -2,6 +2,8 @@ import React, { useState, useEffect } from 'react';
 import { Helmet } from 'react-helmet';
 import Loading from '../../Loading/Loading';
 import { InjectAppServices } from '../../../services/pure-di';
+import logo from './logo.svg';
+import './Shopify.css';
```

Comment:
What do you think about using lowercase by default for the file names?

Reply:
(if you change it, remember to also change the file name)

Reply:
For all? We were using uppercase for components...

Reply:
Oh, I did not understand the logic, it has the same name than the component. I understand it now. Thanks.